src/components/pages/deployments/deploymentsHome/flyouts/deploymentNew/deploymentNew.js

```diff
@@ -129,19 +134,27 @@ export class DeploymentNew extends LinkedComponent {
     );
 
     switch (selectedPackageType) {
-      // case Edge manifest
-      case 'EdgeManifest':
-        const { fetchPackages } = this.props;
-        fetchPackages();
-        this.setState({ edgePackageSelected: true });
+      // case - Edge manifest
+      case packageTypeOptions[0]:
```

Comment:
This feels fragile. Perhaps add constants for each one and a constant containing the array of selections.  Perhaps handle it sort of like permissions.  See authModels.js.   There's an object with properties for each named permission. Then when the array is needed, we use `Object.values(permissions)`

Reply:
done